index.js

```diff
@@ -92,15 +92,14 @@ const drawElement = (_element) => {
 // drawing on a canvas
 const constructLayerToDna = (_dna = [], _layers = [], _rarity) => {
   let mappedDnaToLayers = _layers.map((layer, index) => {
-    let selectedElement = layer.elements[_rarity][_dna[index]];
+    let selectedElement = layer.elements.find(element => element.id === _dna[index]);
```

Comment:
`layer.elements` is now not split into different rarity groups anymore, but contains all elements for a layer. 
In addition, a new property `elementIdsForRarity` was introduced that contains the mapping of rarity to element